presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcConnectorFactory.java

```diff
@@ -67,12 +68,13 @@ public Connector create(String catalogName, Map<String, String> requiredConfig,
         requireNonNull(requiredConfig, "requiredConfig is null");
 
         Bootstrap app = new Bootstrap(
-                binder -> binder.bind(TypeManager.class).toInstance(context.getTypeManager()),
-                binder -> binder.bind(NodeManager.class).toInstance(context.getNodeManager()),
-                binder -> binder.bind(VersionEmbedder.class).toInstance(context.getVersionEmbedder()),
-                new JdbcModule(catalogName),
-                new CredentialProviderModule(),
```

Comment:
Maybe simply let's remove it from here and it via `JdbcConnectorFactory` in all places where it is used instead of this refactor?